estate/models/estate_property.py

```diff
@@ -0,0 +1,87 @@
+from odoo import api, models, fields
+from odoo.exceptions import UserError
+import datetime
+from dateutil.relativedelta import relativedelta
+from odoo.tools.float_utils import float_compare, float_is_zero
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Property for the Real Estate app"
+
+    name = fields.Char(required=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    notes = fields.Html()
+    date_availability = fields.Date(copy=False, default=datetime.date.today() + relativedelta(months=+3))
```

Comment:
```suggestion
    date_availability = fields.Date(copy=False, default=lambda self: datetime.date.today() + relativedelta(months=+3))
```

Use a lambda here, or the version that has been evaluated the first time will always be used.

Reply:
And why `months=+3` ?